WordPress/src/androidTest/java/org/wordpress/android/e2e/flows/SignupFlow.java

```diff
@@ -83,6 +84,11 @@ public SignupFlow dismissInterstitial() {
         return this;
     }
 
+    public SignupFlow dismissJetpackAd() {
+        dismissJetpackAdIfPresent();
+        return this;
+    }
```

Comment:
In order not to break the method chaining in `signUpWithMagicLink` test, and at the same avoid code duplication, I found no better way than to wrap `dismissJetpackAdIfPresent` added to `WPSupportUtils.java` into this function that additionally returns the needed class.